repo2docker/buildpacks/_r_base.py

```diff
@@ -55,8 +55,8 @@ def rstudio_base_scripts():
             "${NB_USER}",
             # Install nbrsessionproxy
             r"""
-                pip install --no-cache-dir https://github.com/jupyterhub/jupyter-server-proxy/archive/7ac0125.zip && \
-                pip install --no-cache-dir jupyter-rsession-proxy==1.0b6 && \
+                pip install --no-cache-dir https://github.com/jupyterhub/jupyter-server-proxy/archive/00be53dfed8ab5a0baaf8332e3fc6bd44dc5bf1a.zip && \
```

Comment:
```suggestion
                pip install --no-cache-dir jupyter-server-proxy==1.3.1 && \
```

Reply:
I applied this but then removed it again because 1.3.1 doesn't include the increased timeout.